5.sc_Morphology_Logistic_Regression/0.split_data/scripts/split_data.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import ast
+import itertools
+import pathlib
+
+import numpy as np
+import pandas as pd
+import pyarrow.parquet as pq
+import toml
+from joblib import dump
+from sklearn.exceptions import ConvergenceWarning
+from sklearn.linear_model import LogisticRegression
+from sklearn.model_selection import GridSearchCV, StratifiedKFold, train_test_split
+from sklearn.utils import parallel_backend, shuffle
+
+# In[2]:
+
+
+# Parameters
+cell_type = "SHSY5Y"
+aggregation = False
+nomic = False
+flag = True
+control = "DMSO_0.100_DMSO_0.025"
+treatment = "Thapsigargin_1.000_DMSO_0.025"
+
+
+# In[3]:
+
+
+print(cell_type, aggregation, nomic, flag, control, treatment)
```

Comment:
A very small thing (feel free to ignore), but I think this may be unnecessary given that all these parameters are defined in the cell above.

Reply:
removing, this was a troubleshooting line